glancesync/glancesync.py

```diff
@@ -65,27 +65,26 @@ def __init__(self, config_stream=None):
         self.targets = glancesyncconfig.targets
         count = 0
         for target in self.targets.values():
-            if 'facade_module' in target:
-                import_code = 'from ' + target['facade_module'] +\
-                              'import ServersFacade as Facade' + str(count)
-                instance_code = 'Facade' + str(count) + '(target)'
-                exec import_code
-                target['facade'] = eval(instance_code)
+            if 'GLANCESYNC_USE_MOCK' in os.environ:
+                target['facade'] = ServersFacadeMock(target)
+            elif 'GLANCESYNC_MOCKPERSISTENT_PATH' in os.environ:
+                target['facade'] = ServersFacadeMock(target)
+                target['facade'].init_persistence(
+                    os.environ['GLANCESYNC_MOCKPERSISTENT_PATH'])
             else:
-                if 'GLANCESYNC_USE_MOCK' in os.environ:
-                    target['facade'] = ServersFacadeMock(target)
-                elif 'GLANCESYNC_MOCKPERSISTENT_PATH' in os.environ:
-                    target['facade'] = ServersFacadeMock(target)
-                    target['facade'].init_persistence(
-                        os.environ['GLANCESYNC_MOCKPERSISTENT_PATH'])
-                else:
-                    target['facade'] = ServersFacade(target)
+                target['facade'] = ServersFacade(target)
 
         self.preferable_order = glancesyncconfig.preferable_order
         self.max_children = glancesyncconfig.max_children
         master_region = GlanceSyncRegion(self.master_region, self.targets)
         images = master_region.target['facade'].get_imagelist(master_region)
-        self.master_region_dict = glancesync_ami.get_master_region_dict(images)
+        # Ignore (public) images of other tenants
```

Comment:
This code should be part of other PR with its own unit tests (different feature). If it is needed, create a new ticket in Jira to describe it.